frappe/email/doctype/auto_email_report/auto_email_report.py

```diff
@@ -207,19 +207,21 @@ def send(self):
 		if not self.format == "HTML":
 			attachments = [{"fname": self.get_file_name(), "fcontent": data}]
 
-		frappe.sendmail(
```

Comment:
After further investigation this feels like wrong fix. 


`@frappe.read_only` should only wrap the function. This function is not read only so the error shouldn't occur here in first place. 

When code reaches this line it should've already swapped to writable connection.

https://github.com/frappe/frappe/blob/d93bf3825112ef6b5ef23cc2a363c1bb10716bbb/frappe/__init__.py#L788-L805